Spec/RealtimeClientConnection.swift

```diff
@@ -3580,18 +3583,86 @@ class RealtimeClientConnection: QuickSpec {
                         }
                         client.connect()
                     }
-
-                    expect(NSRegularExpression.match(testHttpExecutor.requests[0].url!.absoluteString, pattern: "//internet-up.ably-realtime.com/is-the-internet-up.txt")).to(beTrue())
-                    expect(urlConnections).to(haveCount(6)) // default + 5 fallbacks
-
+                    
                     let extractHostname = { (url: NSURL) in
                         NSRegularExpression.extract(url.absoluteString, pattern: "[a-e].ably-realtime.com")
                     }
-                    let resultFallbackHosts = urlConnections.compactMap(extractHostname)
+
+                    var resultFallbackHosts = [String]()
+                    var gotInternetIsUpCheck = false
+                    for url in urls {
+                        if NSRegularExpression.match(url.absoluteString, pattern: "//internet-up.ably-realtime.com/is-the-internet-up.txt") {
+                            gotInternetIsUpCheck = true
+                        } else if let fallbackHost = extractHostname(url) {
+                            if Optional(fallbackHost) == resultFallbackHosts.last {
+                                continue
+                            }
+                            // Host changed; should've had a internet check before.
```

Comment:
Small typo: "should've had a internet"

Reply:
Fixed at 147d7b7a1944243244b0d5b2ea2ee95a2c414781.